src/redux.js

```diff
@@ -971,6 +971,31 @@ export function getEmptyCellDetails(state) {
   return emptyCellLocations;
 }
 
+export function getDataDescription(state) {
+  // If this a dataset from the internal collection that already has a description, use that.
+  if (
+    state.metadata
+    && state.metadata
+    && state.metadata.card
+    && state.metadata.card.description
```

Comment:
Looks like `state.metadata` is repeated. Did you mean for the first one to be `state`? 

I wonder if this would be a good place to use the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining). I happened upon it when trying to figure out something else recently.

Reply:
https://github.com/code-dot-org/code-dot-org/pull/39543 Here' where Charlie enabled chaining in Code Studio

Reply:
Very interesting! Thank you for the reference.